packages/docs/src/theme/ReactLiveScope/index.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import * as PIXI from 'pixi.js';
 import * as ReactPixi from '@pixi/react';
 import * as ReactPixiAnimated from '@pixi/react-animated';
-import { Spring } from 'react-spring';
+import { Spring } from '@react-spring/web';
```

Comment:
For the docs i also used `react-spring/web`. The internet said this was better?

Reply:
Ah for sure! That would also likely remove some npm install errors as some of the other react spring packages have old peer deps